src/client/tsconfig.json

```diff
@@ -1,37 +1,34 @@
 {
   "compilerOptions": {
+    "outDir": "build/dist",
+    "module": "esnext",
     "target": "es5",
-    "module": "commonjs",
-    "moduleResolution": "node",
+    "lib": [
+      "es2015",
+      "es2017",
+      "dom"
+    ],
     "sourceMap": true,
-    "removeComments": true,
+    "allowJs": true,
     "jsx": "react",
+    "moduleResolution": "node",
+    "rootDir": "src",
     "forceConsistentCasingInFileNames": true,
-    "strict": true,
-    "declaration": false,
+    "noImplicitReturns": false,
```

Comment:
Why have we removed strict=true and now do individual settings? I thought the former was ideal, especially in terms of best practices for people looking at the code. Using strict and removing all the settings it covers can simplify the config a lot.

Reply:
Fixed